www/front_src/src/styles/partials/_side-nav.scss

```diff
@@ -437,11 +438,7 @@ a {
       .collapsed-level-item {
         background-color: $white-color;
         .collapsed-level-item-link {
-          padding: 8px 10px 8px 47px;
-          // &:hover {
-          //   color: $white-color;
-          //   background-color: #0a78a5;
-          // }
+          padding: 8px 10px 8px 56px;

```

Comment:
this modification is from another PR not yet approved by the QA : https://github.com/centreon/centreon/pull/7251